addons/sale_timesheet/models/project.py

```diff
@@ -591,15 +604,16 @@ def _get_last_sol_of_customer(self):
         if not self.partner_id.commercial_partner_id or not self.allow_billable:
             return False
         domain = [
-            ('company_id', '=', self.company_id.id),
             ('is_service', '=', True),
             ('order_partner_id', 'child_of', self.partner_id.commercial_partner_id.id),
             ('is_expense', '=', False),
             ('state', '=', 'sale'),
-            ('remaining_hours', '>', 0),
+            ('remaining_hours', '>', 0)
         ]
         if self.project_id.pricing_type != 'task_rate' and self.project_sale_order_id and self.partner_id.commercial_partner_id == self.project_id.partner_id.commercial_partner_id:
             domain.append(('order_id', '=?', self.project_sale_order_id.id))
+        if self.company_id:
+            domain.append(('company_id', '=', self.company_id.id))
```

Comment:
Can be replaced by `('company_id', '=?', self.company_id.id)` directly in the domain list if you want. 🙂